apps/condo/migrations/20220603103829-0135_propertyhistoryrecord_maphash.js

```diff
@@ -0,0 +1,26 @@
+// auto generated by kmigrator
+// KMIGRATOR:0135_propertyhistoryrecord_maphash:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMC40IG9uIDIwMjItMDYtMDMgMDU6MzgKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zLCBtb2RlbHMKCgpjbGFzcyBNaWdyYXRpb24obWlncmF0aW9ucy5NaWdyYXRpb24pOgoKICAgIGRlcGVuZGVuY2llcyA9IFsKICAgICAgICAoJ19kamFuZ29fc2NoZW1hJywgJzAxMzRfYXV0b18yMDIyMDYwMl8wODU1JyksCiAgICBdCgogICAgb3BlcmF0aW9ucyA9IFsKICAgICAgICBtaWdyYXRpb25zLkFkZEZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdwcm9wZXJ0eWhpc3RvcnlyZWNvcmQnLAogICAgICAgICAgICBuYW1lPSdtYXBIYXNoJywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLkpTT05GaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICBdCg==
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Add field mapHash to propertyhistoryrecord
+--
+ALTER TABLE "PropertyHistoryRecord" ADD COLUMN "mapHash" jsonb NULL;
```

Comment:
if we would search by this field, it'd be better to add an index

Reply:
no, it's History %)